examples/next-prisma-starter-sqlite/src/server/routers/health.ts

```diff
@@ -1,5 +1,5 @@
-import { createRouter, baseProcedure } from '../trpc';
+import { router, baseProcedure } from '../trpc';
```

Comment:
In docs you've changed the name of the `baseProcedure` to just `procedure`, or in some cases `publicProcedure`. Should prob keep the examples consistent.

Reply:
Out for lunch now, feel free to poke around

I think best would be publicProcedure everywhere